### DISCUSSION TITLE
Chia is gone

### DISCUSSION THREAD

**Discussion by Raaajup** [Chia-Network/chia-blockchain#5339] [Support]
Hi Guys,

My wallet the other day was stuck "connecting to wallet", so in the end I deleted .chia file from user folder on c drive and reinstalled chia client (I did copy the file so still have it). I then entered my key which had farmed a few Chia and let it sync back up, however once it had synced it still says my balance is 0 having synced for over a day. Is there any way of getting the Chia that I have farmed back?   

Thank you any help really appreciated.

**Comment by avitron78** [comment]
Check that these three things are in sync. 1 "connection status", 2 "Status", 3 "Wallet" You should have all three things in sync. That was what they told me from the technical support, they told me that the wallet also had to synchronize and since in my case it was a new wallet, it had not synchronized, I have been waiting almost two days for the wallet to synchronize, the numbers advance very slowly, one Once I do, I will check if my issue has been resolved.

**Comment by Raaajup** [comment]
Thank you I'll give it a go and try to be patient..

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
